CHANGELOG.md

```diff
@@ -77,8 +77,9 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Changed
 
+- SC-9192 - mount feathers and nestjs apps under dedicated version paths and allow general path prefix for the whole server
```

Comment:
Is this line at the right place on the change log?

Reply:
will be moved to the top